packages/shared/src/components/feeds/FeedContainer.tsx

```diff
@@ -157,21 +155,16 @@ export const FeedContainer = ({
   });
   const currentSettings = useContext(FeedContext);
   const { subject } = useToastNotification();
-  const {
-    spaciness,
-    insaneMode: listMode,
-    loadedSettings,
-  } = useContext(SettingsContext);
-  const { shouldUseListFeedLayout, isListModeV1 } = useFeedLayout();
+  const { spaciness, loadedSettings } = useContext(SettingsContext);
+  const { shouldUseListFeedLayout, isListMode } = useFeedLayout();
   const isLaptop = useViewSize(ViewSize.Laptop);
   const { feedName } = useActiveFeedNameContext();
   const router = useRouter();
   const numCards = currentSettings.numCards[spaciness ?? 'eco'];
-  const insaneMode = !forceCardMode && listMode;
   const isList =
-    (isHorizontal || isListModeV1) && !shouldUseListFeedLayout
+    (isHorizontal || isListMode) && !shouldUseListFeedLayout
       ? false
-      : (insaneMode && numCards > 1) || shouldUseListFeedLayout;
+      : (isListMode && numCards > 1) || shouldUseListFeedLayout;
```

Comment:
I got the feeling this can now also be simplified as it contained logic for old list mode - but will need to check.

Reply:
Alright will leave it for this task as It's already a bit much.
Do think we can as we have also stuff like this: (which I think just is the same thing twice 😅)

```
      gapClass({
        isList,
        isFeedLayoutList: shouldUseListFeedLayout,
```